nbs/03_matcher.ipynb

```diff
@@ -10,7 +10,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.        for node in collection_pattern.nodes:
</pre><p>simply use set intersection here</p><p> </p>

<pre class="ql-syntax" spellcheck="false">set_1 &amp; set_2
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DeanLight/graph_rewrite/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>done, thank you</p><p> </p>